docs/sphinx/source/api.rst

```diff
@@ -21,6 +21,8 @@ corresponding procedural code.
    location.Location
    pvsystem.PVSystem
    pvsystem.Array
+   pvsystem.FixedMount
+   pvsystem.SingleAxisTrackerMount
```

Comment:
perhaps we should link to the pvsystem.rst and modelchain.rst in the intro text to this table.